src/minmax.rs

```diff
@@ -0,0 +1,90 @@
+use rustc::lint::{Context, LintPass, LintArray};
+use rustc_front::hir::*;
+use syntax::codemap::Spanned;
+use syntax::ptr::P;
+use std::cmp::PartialOrd;
+use std::cmp::Ordering::*;
+
+use consts::{Constant, constant};
+use utils::{match_path, span_lint};
+use self::MinMax::{Min, Max};
+
+declare_lint!(pub MIN_MAX, Deny,
```

Comment:
> Deny lints should be perfect.

I think Deny lints may have no false positives on code that was written deliberately. They may have false negatives, however. Would you still insist on Warn if I removed all possible false positives?